R/zchunk_L1233.Elec_water.R

```diff
@@ -39,89 +38,301 @@ module_water_L1233.Elec_water_DISABLED <- function(command, ...) {
     iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
     calibrated_techs <- get_data(all_data, "energy/calibrated_techs")
     enduse_fuel_aggregation <- get_data(all_data, "energy/enduse_fuel_aggregation")
-    L101.en_bal_EJ_ctry_Si_Fi_Yh_full <- get_data(all_data, "L101.en_bal_EJ_ctry_Si_Fi_Yh_full")
-    L1231.in_EJ_R_elec_F_tech_Yh <- get_data(all_data, "L1231.in_EJ_R_elec_F_tech_Yh")
-    L1231.out_EJ_R_elec_F_tech_Yh <- get_data(all_data, "L1231.out_EJ_R_elec_F_tech_Yh")
     A23.CoolingSystemShares_RG3 <- get_data(all_data, "water/A23.CoolingSystemShares_RG3")
     elec_tech_water_map <- get_data(all_data, "water/elec_tech_water_map")
     Macknick_elec_water_m3MWh <- get_data(all_data, "water/Macknick_elec_water_m3MWh")
 
+    L101.en_bal_EJ_ctry_Si_Fi_Yh_full <-
+      get_data(all_data, "temp-data-inject/L101.en_bal_EJ_ctry_Si_Fi_Yh_full") %>%
```

Comment:
I think if you merge `master` this is now available; change and remove from `temp-data-inject` please